docs/blog.md

```diff
@@ -316,6 +316,10 @@ username 为博主用户名，而非`xxx.hashnode.dev`中`xxx`所代表的 blog
 
 ## 十年之约
 
+### 专题展示-文章
+
+<Route author="a180285" example="foreverblog-cn/feeds" path="foreverblog-cn/feeds" radar="1" rssbud="1" />
```

Comment:
Please use path `/oreverblog`

Reply:
This needs to be addressed before merging

Reply:
Changed, I misunderstand that you ask me to add a slash at the begining.